lib/ansible/modules/packaging/os/portage.py

```diff
@@ -326,13 +326,20 @@ def emerge_packages(module, packages):
         module.fail_json(msg='Use only one of usepkg, usepkgonly')
 
     emerge_flags = {
-        'jobs': '--jobs=',
-        'loadavg': '--load-average ',
+        'jobs': '--jobs',
+        'loadavg': '--load-average',
     }
 
     for flag, arg in emerge_flags.items():
-        if p[flag] is not None:
-            args.append(arg + str(p[flag]))
+        flag_val = p[flag]
+        if flag_val is None or flag_val is False:
```

Comment:
so the 'is' will only work with boolean values so you cannot set a jobs = 5 nor jobs = '', the original was not completely correct either:

```
if p[flag]:
   args.append(arg + to_native(p[flag])
```
^ this should work for all cases

Reply:
yes, I just left it copy-pasted for now. will fix

Reply:
I'm still not sure about public API here. Is it possible to make schema accept a truthful boolean or a number in one place?

Reply:
yes, type=raw will avoid type enforcement, but it is not normally a good interface

Reply:
What would you advise given that these options can be invoked w/o param?  (https://github.com/ansible/ansible/pull/36355#issuecomment-366482579)

Reply:
probably keep as int and take 0/false values as removing the previous setting.

```
if val:
   append flag + val
elif val is not None:
   append flag

Reply:
Will the type checker allow user bypassing `False` into that int value? Or you mean it's okay to introduce special behavior for `int(0)` since it's being `bool`eaned as `False`?